### PR TITLE
Giving local storage to the form

### PR DESCRIPTION
**Pull request by Aliahmad-karimi**
- Add input values to the storage.
- fill the input values with the storage values.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -140,4 +140,47 @@ document.addEventListener('DOMContentLoaded', () => {
...
+  function populateStorage() {
+    storageForm.fName = firstName.value;
+    storageForm.fEmail = emailAdd.value;
+    storageForm.fMsg = sms.value;
+    localStorage.setItem('data', JSON.stringify(storageForm));
+  }
+
+  function getFormStorage() {
+    const reFill = JSON.parse(localStorage.getItem('data'));
+    firstName.value = reFill.fName;
```
Comment (HAMMAS-SALEEM) [inline script.js:163]: ### Your Form and Error
error

- [x] Your form is not setting and getting data from localStorage. Please fix this issue :+1:\

File: script.js [hunk 2]
```diff
@@ -140,4 +140,47 @@ document.addEventListener('DOMContentLoaded', () => {
...
+    sms.value = reFill.msg;
+  }
+
+  function setDataFirst() {
+    if (localStorage.getItem('data') === null) {
+      localStorage.setItem('data', JSON.stringify(storageForm));
+    }
+  }
+
+  getFormStorage();
```
Comment (HAMMAS-SALEEM) [inline script.js]: - [x] You should run this function if localStorage has data. :thinking: You can use

```
    if (localStorage.getItem('data') !== null) {
      getFormStorage();
    }
```

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @Aliahmad-karimi, @osbentech :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Descriptive PR title
:heavy_check_mark: Descriptive PR summary
:heavy_check_mark: Descriptive Branch name
:heavy_check_mark: Following Linters

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:


# Optional suggestions
_See Comments Below_

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Aliahmad-karimi 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- No Linter errors ✅
- Your repo is professional ✅
- Well structured files ✅



Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**